### PR TITLE
Fix glyphicons

### PR DESCRIPTION
**Pull request by langjoh**

### CODE REVIEW

File: app/assets/stylesheets/modules/_buttons.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  width: 140px;
+  position: absolute;
+  margin-top: -72px;
+  top: 50%;
+  left: 25%;
+  margin-left: -70px;
+}
+
+@media (max-width: 1140px) {
+
```
Comment (berlintam) [inline app/assets/stylesheets/modules/_buttons.scss]: Is this needed/used anywhere?

File: app/assets/stylesheets/modules/_buttons.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  margin-left: -80px;
+  text-align: left;
+}
+
+.control-btn {
+  cursor: pointer;
+}
+
+.play-button.control-btn .glyphicon {
+  color: #ff7400;
```
Comment (berlintam) [inline app/assets/stylesheets/modules/_buttons.scss]: can you use the colour variables here instead?

File: app/assets/stylesheets/modules/_buttons.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+.control-btn .glyphicon {
+  color: #F9A120;
```
Comment (berlintam) [inline app/assets/stylesheets/modules/_buttons.scss]: here as well?

File: app/assets/stylesheets/modules/_container.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    position: relative;
+    left: 0;
+    margin-left: 0;
+    top: 0;
+    margin-top: 0;
+  }
+}
+
+@media (max-width: 1140px) {
+ .drop-container{
```
Comment (berlintam) [inline app/assets/stylesheets/modules/_container.scss:21]: it seems like you have two `.drop-container` classes here... (also one on line 11). Is that on purpose? will one over-ride the other? (I'm asking b/c I'm not a css pro & don't really understand/know what's meant to happen here. I'm learning too!)

**Comment by berlintam** [comment]
hey! This is awesome! I just made a few comments (mostly due to my own lack of understanding of all things css), but it's DEFINITELY a lot better. 

I pulled the branch & the tracks aren't playing for me.... The glyphicons are definitely a lot better, but the tracks don't play when I hit play. Do they work for you?

File: app/assets/stylesheets/modules/_container.scss (hunk 4 above)
Comment (langjoh) [inline app/assets/stylesheets/modules/_container.scss:21]: on line 10 starts a media query. It defines what the .drop-container class should behave like, if the maximum **height** of the container is 870px. In line 20 starts another media query that defines what the same class should behave like, if the container has a maximum **width** of 1140px. So the two media query don't overwrite each other, they are like two if statements for different cases, one for max height and one for max width.

Comment (berlintam) [inline app/assets/stylesheets/modules/_container.scss:21]: ah, gotcha! thanks!

**Pull request merged**